framework/ui/view/musicalsymboliconcodes.h

```diff
@@ -0,0 +1,70 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+#define MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+
+#include <QObject>
+
+namespace mu {
+namespace framework {
+/**
+ * @brief The MusicalSymbolIconCode class simplifies access to the icons from the musical icon font
+ *
+ * @details Each enum value is a UTF-16-like address of the icon in the icon font.
+ */
+
+class MusicalSymbolIconCode
+{
+    Q_GADGET
+
+public:
+    enum class Code : char16_t {
+        ZERO = 0xE080,
```

Comment:
Do we really need these codes?
Isn't it enough to just use the exact numbers ("0", "1", "2", etc.) in our QML labels?

Reply:
No, because notation fonts depend on the SMuFL format. The SMuFL format expands the Unicode, but not rewrite, it is the standard. Therefore, we cannot use only numbers, because we will not find glyphs in notation fonts.